### ISSUE TITLE
WARNING: The examine memory location command won't work

### ISSUE THREAD

**Issue by boyedarat** [WebFreak001/code-debug#78]
What do these warnings mean?
Code-Debug WARNING: Utility Command Server: Error in command socket Error: listen EACCES C:\Users\st4r\AppData\Local\Temp\code-debug-sockets\Debug-Instance-n8sl
Code-Debug WARNING: The examine memory location command won't work

I get this warning when debugging a Rust program using GDB. Is this related to this enhancement?:
https://github.com/WebFreak001/code-debug/issues/64

**Comment by WebFreak001** [comment]
well these warnings mean that "The examine memory location command won't work"

Otherwise you can just ignore them. I should fix this on windows

**Comment by Lin-H** [comment]
I have the same problem with Rust debugging. It's not just warning. 
An error occur `Debug adapter process has terminated unexpectedly`
So it couldn't work

VSCode 1.6.1
GDB 7.12
Rust 1.12.1

**Comment by WebFreak001** [comment]
no it most likely doesn't have anything to do with that crash, try checking out #59. Otherwise does it print anything to the console before crashing other than that warning?

**Comment by Lin-H** [comment]
I figured it out ! First I just put gdb.exe in the PATH, I can use it in the command line, but maybe it's not enough. So I put the whole folder `E:\msys64\mingw64\bin` in the PATH. 
IT WORDED !
Thank you for your help @WebFreak001 . I thought the plugin crushed at first.

**Comment by ctaggart** [comment]
I also get the error when trying to debug and I can't seem to hit a breakpoint.

```
Code-Debug WARNING: Utility Command Server: Error in command socket Error: listen EACCES C:\Users\cameron\AppData\Local\Temp\code-debug-sockets\Debug-Instance-mjhs
Code-Debug WARNING: The examine memory location command won't workNo symbol table is loaded.  Use the "file" command.
No symbol table is loaded.  Use the "file" command.
```

Latest installed:
```
PS C:\Users\cameron\rs\app1> cmd /c where gdb
C:\msys64\mingw64\bin\gdb.exe
PS C:\Users\cameron\rs\app1> gdb --version
GNU gdb (GDB) 7.12
PS C:\Users\cameron\rs\app1> rustc --version
rustc 1.12.1 (d4f39402a 2016-10-19)
PS C:\Users\cameron\rs\app1> code --version
1.7.1
02611b40b24c9df2726ad8b33f5ef5f67ac30b44
```

![image](https://cloud.githubusercontent.com/assets/80104/20027342/b46e6db6-a2ce-11e6-858a-e311c8b7a32f.png)

**Comment by ctaggart** [comment]
Ugh, for me it looks like the important error is:
```
No symbol table is loaded.  Use the "file" command.
```

And this is probably because I'm using the wrong toolchain:
```
PS C:\Users\cameron\rs\app1> cmd /c where rustc
C:\Program Files\Rust stable MSVC 1.12\bin\rustc.exe
```

I'll get some sleep and fix that. Sorry to bother.

**Comment by AprliRainkun** [comment]
I have the same problem with Rust debugging on windows. I also found that when I hovered my cursor on a variable with complex type (for example a string Iterator ) , the debugger would crash. While the primitive types would show there content correctly. 

VS Code: 1.7.1
Rust: GNU 1.12
GDB: 7.9.1
![qq 20161108170933](https://cloud.githubusercontent.com/assets/15924210/20093119/5b452738-a5d6-11e6-9b1e-63bf756275d2.png)
